dht/range_streamer.cc

```diff
@@ -222,10 +222,16 @@ range_streamer::get_all_ranges_with_strict_sources_for(const sstring& keyspace_n
 bool range_streamer::use_strict_sources_for_ranges(const sstring& keyspace_name) {
     auto& ks = _db.local().find_keyspace(keyspace_name);
     auto& strat = ks.get_replication_strategy();
-    return !_db.local().is_replacing()
+    auto rf = strat.get_replication_factor();
+    auto nr_nodes_in_ring = get_token_metadata().get_all_endpoints().size();
```

Comment:
Continuing my previous question - what's the difference between get_all_endpoints().size() and get_all_endpoints_count()? Can you get of the _count() function if you don't use it?

Reply:
We do have a user of get_all_endpoints_count. After this patch get_all_endpoints_count and get_all_endpoints.size() are the same. But get_all_endpoints_count is a bit more efficient than get_all_endpoints.size() due to the implementation.

Reply:
If get_all_endpoints_count() is more efficient, why doesn't get_all_endpoints() use the same efficient implementation?